Entities/Voter.cs

```diff
@@ -0,0 +1,14 @@
+namespace Olivia.Entites;
+
+class Voter
+{
+    public int Id { get; set; }
+    public string NIM { get; set; }
```

Comment:
Should be named with english, ie : NIM (Nomor Induk Mahasiswa) become SIN (Student ID Number)

Reply:
Sudah om silakan di cek